### ISSUE TITLE
Implement codeview sidepanel

### ISSUE THREAD

**Issue by sacha-l** [substrate-developer-hub/substrate-docs#272]
Create a plugin to have a sidepanel codeview in some how-to guides.

**Comment by jonnysmillie** [comment]
Just want to clarify this issue @sacha-l  @NukeManDan. Is this an issue to create a layout that allows for a longer codeblock in the right hand column as on Stripe.com docs?

Also is it still actively needed?

![Screenshot 2022-06-27 at 16 38 32](https://user-images.githubusercontent.com/7614348/175979012-3ce15273-f2da-4586-bfc1-acb61ce8a1d1.png)

**Comment by sacha-l** [comment]
> Just want to clarify this issue @sacha-l @NukeManDan. Is this an issue to create a layout that allows for a longer codeblock in the right hand column as on Stripe.com docs?

Yes, the basic reference example of how Substrate developer hub efforts previously had something like this would be: https://www.shawntabrizi.com/substrate-collectables-workshop/#/1/storage-mapping - where users can toggle between the "potential solution", "diff" and "full solution", for example of how this can be useful.
